tests/integration/database/migrations/issuer-field-migration.test.ts

```
@@ -0,0 +1,267 @@
+      `);
+    });
+
+    afterEach(async () => {
+      if (client) {
+        await client.end();
+      }
+    });
+
+    it('should add issuer_id column to assertions table', async () => {
+      if (!process.env.DATABASE_URL) {
+        console.log('Skipping PostgreSQL test - DATABASE_URL not set');
```

ESLint

[error] Unexpected console statement.

(no-console)